robot/pmm/tests/browser/Attendance/dim_attendance.robot

```diff
@@ -19,7 +19,7 @@ Setup Test Data
     Set suite variable              ${ns}
     ${contact1}                     API Create Contact
     Set suite variable              ${contact1}
-    ${contact2}                     API Create Contact      FirstName=test robot contact
+    ${contact2}                     API Create Contact      FirstName=ztest robot contact
```

Comment:
If you don't really care what the contact name is, don't forget that you can use our [faker](https://confluence.internal.salesforce.com/display/SFDOIDP/How+to+Create+Fake+Test+Data+with+Faker) support to generate names for you.